x-pack/plugins/security_solution/public/management/components/console/components/command_execution_result.tsx

```diff
@@ -76,6 +84,19 @@ export const CommandExecutionResult = memo<CommandExecutionResultProps>(
       });
     }, [className, showAs]);
 
+    const titleMessage = useMemo(() => {
+      if (title) {
+        return title;
+      }
+      if (showAs === 'success') {
+        return agentType === 'crowdstrike'
```

Comment:
nit: maybe make a `crowdstrike` const somewhere, we seem to be using this in a lot of places.